examples/oft-solana/tasks/solana/createOFTAdapter.ts

```diff
@@ -35,33 +48,57 @@ interface CreateOFTAdapterTaskArgs {
 }
 
 // Define a Hardhat task for creating OFTAdapter on Solana
-task('lz:oft-adapter:solana:create', 'Mints new SPL Token, Lockbox, and new OFT Adapter Store account')
+task('lz:oft-adapter:solana:create', 'Creates new OFT Adapter (OFT Store PDA)')
     .addParam('mint', 'The Token Mint public key')
-    .addParam('programId', 'The OFT Program id')
+    .addParam('programId', 'The OFT program ID')
     .addParam('eid', 'Solana mainnet or testnet', undefined, devtoolsTypes.eid)
     .addParam('tokenProgram', 'The Token Program public key', TOKEN_PROGRAM_ID.toBase58(), devtoolsTypes.string, true)
+    .addParam('additionalMinters', 'Comma-separated list of additional minters', undefined, devtoolsTypes.csv, true)
+    .addOptionalParam(
+        'onlyOftStore',
+        'If you plan to have only the OFTStore and no additional minters.  This is not reversible, and will result in losing the ability to mint new tokens by everything but the OFTStore.',
+        false,
+        devtoolsTypes.boolean
+    )
     .setAction(
         async ({
+            additionalMinters: additionalMintersAsStrings,
             eid,
             mint: mintStr,
+            onlyOftStore,
             programId: programIdStr,
             tokenProgram: tokenProgramStr,
         }: CreateOFTAdapterTaskArgs) => {
             const { connection, umi, umiWalletKeyPair } = await deriveConnection(eid)
             const { programId, lockBox, escrowPK, oftStorePda, eddsa } = deriveKeys(programIdStr)
+            if (!additionalMintersAsStrings) {
+                if (!onlyOftStore) {
+                    throw new Error('If you want to proceed with only the OFTStore, please specify --onlyOFTStore')
+                }
+                console.log(
+                    'No additional minters specified.  This will result in only the OFTStore being able to mint new tokens.'
+                )
+            }
+            const additionalMinters = additionalMintersAsStrings?.map((minter) => new PublicKey(minter)) ?? []
             const tokenProgram = publicKey(tokenProgramStr)
             const mint = publicKey(mintStr)
             const mintAuthorityPublicKey = await createMintAuthorityMultisig(
                 connection,
                 toWeb3JsKeypair(umiWalletKeyPair),
                 toWeb3JsPublicKey(oftStorePda),
-                toWeb3JsPublicKey(tokenProgram)
+                toWeb3JsPublicKey(tokenProgram),
+                additionalMinters
             )
+            console.log(`created SPL multisig @ ${mintAuthorityPublicKey.toBase58()}`)
+            await checkMultisigSigners(connection, mintAuthorityPublicKey, [
+                toWeb3JsPublicKey(oftStorePda),
+                ...additionalMinters,
+            ])
```

Comment:
Why is this needed for OFTAdapter?  OFTAdapter is meant to be lock/unlock.  If folks want to modify their mint to support SPL multisig, that is outside of LayerZero scope.

Reply:
Oups, I thought it's needed but it seems it's only needed for logging stuff. Will remove. 😅